etc/conf/server.properties

```diff
@@ -12,6 +12,8 @@ server.token-url=
 server.client-id=
 server.client-secret=
 server.redirect-port=
+# D-Days H-Hours M-Minutes S-Seconds
+server.cookie-timeout=5D
```

Comment:
Lets just require the full syntax string. See my other comment. You can reference `Duration.parse()` in the comment above for context.